build-tools/automation/yaml-templates/install-signing-plugin.yaml

```diff
@@ -0,0 +1,57 @@
+parameters:
+  signType: $(MicroBuildSignType)
+  condition: succeeded()
+
+steps:
+- powershell: |
+    # We require at a minimum version 2.8.0 of the azure cli installed for the ESRP signing
```

Comment:
FYI. As of yesterday, this step was put into the templates repo.  If you wanted to start using these steps from there.

azure-tools/az-client-update.yml

Reply:
Good to know, I've updated this.